.github/workflows/linux_post_commit.yml

```diff
@@ -67,6 +67,11 @@ jobs:
        run: |
          python3 $GITHUB_WORKSPACE/src/buildbot/check.py -w  $GITHUB_WORKSPACE \
          -s  $GITHUB_WORKSPACE/src -o $GITHUB_WORKSPACE/build -t check-llvm-spirv
+     - name: check-xptifw
+       if: always()
+       run: |
+         python3 $GITHUB_WORKSPACE/src/buildbot/check.py -w  $GITHUB_WORKSPACE \
+         -s  $GITHUB_WORKSPACE/src -o $GITHUB_WORKSPACE/build -t check-xptifw
```

Comment:
How long does it take to run these tests?
NOTE: CCache doesn't always improve job's running time, so we need to make sure we do not go above GitHub limits.